src/ESUGApp-Component/ERItemNav.class.st

```diff
@@ -10,7 +10,9 @@ Class {
 	#instVars : [
 		'callback',
 		'icon',
-		'label'
+		'label',
+		'hasNotif',
```

Comment:
This variable was added but I have the impression it is not used. To know if notif is enabled or not you're checking if the count is at 0 or not. So there is no need for this second variable I guess?

Reply:
I did it that way because having 0 notif doesn't mean necessarily the NavItem can't show notifications. 
I wanted to explicitly code the ability to show a notif.

Reply:
`#hasNotif` is not referenced outside the `#initialization`, `#desactivateNotif` and `#activateNotif`

I guess that if you want to display notif for 0 you need to update the rendering method

Reply:
I use the variable in the initialize where it is set to false. when the proposal navItem is being instantiated I call the activateNotif(which set hasNotif to true) method  on it.

Reply:
It is indeed set but then it is never used. 

Either the fact that it is enabled or not will not be used and in that case, we can remove that to simplify the code (it is in general not a good idea to keep code that does nothing because it makes the maintenance more complicated). 
Or it should display something and in that case, it needs to be read in the rendering method

Reply:
Yes, You're right. The if condition in the rendering is what really decides.

Reply:
What is the expected result you want?

- If the notif is disable and the count > 0?
- If the notif is enable and the count >0?
- If the notif is disable and the count = 0?
- If the notif is enable and the count =0?

Reply:
If the notif is enable and the count >0

Reply:
There is another method that should me modified also. 

`notifCount: aNumber 
	notifCount:=aNumber.
	aNumber > 0 ifTrue: [ self activateNotif ] ifFalse: [ self desactivateNotif  ]`
The line `aNumber > 0 ifTrue: [ self activateNotif ] ifFalse: [ self desactivateNotif  ]` must be deleted then.

Reply:
Hello, what should I do next ? Correct it locally and push it after or ...

Reply:
In general, the best thing to do is to ensure your branch is up to date (you did some changes directly on github that are probably not on your local repository. To do that you can fetch then pull if needed.

Then you can do your changes locally and push them. 

A PR is linked to a branch. If the branch is updated after a push, the PR will automatically be updated

Reply:
@jccfd thanks. I've just pushed the correction. Deleting the temporary variable(`notifs:= self notifCount`) gave an issue since it was used to set the badge value. Instead of calling the `self notifCount ` thrice, I reintroduce `notifs` .